platform/openide.windows/src/org/openide/windows/Mode.java

```diff
@@ -124,4 +124,19 @@
     /** Gets selected <code>TopComponent</code> in this mode.
      * @since 4.13 */
     public TopComponent getSelectedTopComponent();
+    
+    /**
+     * Extension to provide exposure of XML configuration.
+     * 
+     * @see ModeXml
+     * @see http://wiki.apidesign.org/wiki/ExtendingInterfaces
+     */
```

Comment:
Don't forget to add `@since 33.44`...

Reply:
@phipma It seems that you overlook this suggestion.

Reply:
@junichi11 yes - I have added that tag and corrected the comment - ModeXml no longer exists - refactored since then.